Project01/Program.cs

```diff
@@ -1,2 +1,20 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using  Lesson05;
+
+Console.WriteLine(" проверки операторов:");
+
+
+//var t = new RationalNumber(2, 3);
+
+
+RationalNumber[] rns = { new RationalNumber(2, 3), new RationalNumber(-1, 5), new RationalNumber(3, 1) };
+
+foreach (RationalNumber r in rns)
+{
+
+    RationalNumber r1 = rns[0];
+    RationalNumber r2 = r;
+    Console.WriteLine($" операции с {r1.ToString()}   и   {r.ToString()}");
```

Comment:
```suggestion
    Console.WriteLine($" операции с {r1}   и   {r}");
```
В `ToString` нет необходимости